### PR TITLE
Fix 4270 Add icons for GeoStory

### PR DESCRIPTION
**Pull request by allyoucanmap**
## Description
This PR add icons for GeoStory:
- [x] GeoStory: glyph name `geostory`
- [x] Title Section: glyph name `story-title-section`
- [x] Paragraph Section: glyph name `story-paragraph-section`
- [x] Media Section: glyph name `story-media-section`
- [x] Immersive Section: glyph name `story-immersive-section`
- [x] Immersive Content: glyph name `story-immersive-content`
- [x] Fit Cover: glyph name `fit-cover`
- [x] Fit Contain: glyph name `fit-contain`
- [x] Small Size: glyph name `size-small`
- [x] Medium Size: glyph name `size-medium`
- [x] Large Size: glyph name `size-large`
- [x] Full Size: glyph name `size-extra-large`
- [x] Height View: glyph name `height-view`
- [x] Height Auto: glyph name `height-auto`

There is also a new **script** with this PR:

`npm run generate-icons`

This script creates font files (eot, svg, ttf, woff and woff2) from a folder of svg files and it will replace the icons.less file with updated classes.
Note: the name of the svg file will become the name of the class for the glyphicon.
Glyphs from previous font have been exported as svg files in the current folder:
`web\client\themes\default\svg`

## Issues
 - #4270

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)

 - [x] Other... Please describe: new icons and related script

**What is the current behavior?** (You can also link to an open issue here)
Missing icon for GeoStory

**What is the new behavior?**
add icon for GeoStory see description

**Does this PR introduce a breaking change?** (check one with "x", remove the other)

 - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: build/generateIcons.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        cssDest: cssDest,
+        fontName: 'icons',
+        templateOptions: {
+            classPrefix: 'glyphicon-',
+            baseSelector: '.glyphicon'
+        },
+        cssTemplate: cssTemplate
+    }, function(error) {
+        if (error) {
+            console.log(error);
```
Comment (MV88) [inline build/generateIcons.js]: maybe i would log the error after the title

or you can add something to the end

File: build/generateIcons.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            console.log(' ');
+            console.log(' -------------------------------------------------------------------------------');
+            console.log('  ICONS GENERATOR - SUCCESS');
+            console.log(' -------------------------------------------------------------------------------');
+            console.log(' ');
+
+        }
+    });
+} catch (error) {
+    console.log(error);
```
Comment (MV88) [inline build/generateIcons.js]: same thing as above

File: build/generateIcons.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
+
+try {
+    console.log(' ');
+    console.log(' -------------------------------------------------------------------------------');
+    console.log('  ICONS GENERATOR - PROCESS START');
+    console.log(' -------------------------------------------------------------------------------');
+    console.log(' ');
+    const webfontsGenerator = require('vusion-webfonts-generator');
```
Comment (MV88) [inline build/generateIcons.js]: can you use es6 syntax  for imports?

Comment (allyoucanmap) [reply build/generateIcons.js]: I'm using require because some versions of node doesn't not support `import`

**Pull request merged**